plugins/opensource/table.md

```diff
@@ -427,6 +427,138 @@ tinymce.init({
 });
 ```
 
+### `table_border_widths`
+
+{{site.requires_5_9v}}
+
+This option is used to specify a list of pre-defined cell border widths for quick access on the `tablecellborderwidth` toolbar button or menu item. This option accepts any valid css numberic value. [MDN web docs - Numeric data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#numeric_data_types)
+
+**Type:** `Array`
+
+**Default Value:** 
+```
+[
+  {title: '1px', value: '1px'},
+  {title: '2px', value: '2px'},
+  {title: '3px', value: '3px'},
+  {title: '4px', value: '4px'},
+  {title: '5px', value: '5px'}
+]
+```
+
+#### Example: Using `table_border_widths`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'table',
+  menubar: 'table',
+  toolbar: 'table',
+  table_border_widths: [
+    {title: '1 pixel', value: '1px'},
+    {title: '2 pixels', value: '2px'},
+    {title: '3 pixels', value: '3px'},
+    {title: '5 pixels', value: '5px'}
+  ]
+});
+```
+
+### `table_border_styles`
+
+{{site.requires_5_9v}}
+
+This option is used to specify a list of pre-defined cell border widths for quick access on the `tablecellborderstyle` toolbar button or menu item, in addition to the dialog options. This option accepts any valid css borders style. [MDN web docs - Border style](https://developer.mozilla.org/en-US/docs/Web/CSS/border-style#values) 
+
+**Type:** `Array`
+
+**Default Value:** 
+```
+[
+  {title: 'Solid', value: 'solid'},
+  {title: 'Dotted', value: 'dotted'},
+  {title: 'Dashed', value: 'dashed'},
+  {title: 'Double', value: 'double'},
+  {title: 'Groove', value: 'groove'},
+  {title: 'Ridge', value: 'ridge'},
+  {title: 'Inset', value: 'inset'},
+  {title: 'Outset', value: 'outset'},
+  {title: 'None', value: 'none'},
+  {title: 'Hidden', value: 'hidden'}
+]
+```
+
+#### Example: Using `table_border_styles`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'table',
+  menubar: 'table',
+  toolbar: 'table',
+  table_border_styles: [
+    {title: 'Solid', value: 'solid'},
+    {title: 'Dotted', value: 'dotted'},
+    {title: 'Dashed', value: 'dashed'}
+  ]
+});
+```
+
+### `table_cell_background_color_map`
+
+{{site.requires_5_9v}}
+
+This option is used to specify the default values for the table cell background color picker `tablecellbackgroundcolor`. If no values are defined, the toolbar button and menu item will use the values or default values of the [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map). The option can handle Hex, sRGB and HSL, and ignores alpha values.
+
+Custom colors can't be enabled.
```

Comment:
"Custom colors can't be enabled."

what is meant by this?  I assume we can configure any valid hex/sRGB, HSL color to the map and call it anything we like?

Reply:
Yes, you can configure any valid color. However, when using the colorswatch the user can't select custom colors of their own choice, as they can with fontcolors and background color. This is due to a techdebt we intend to solve later.

Reply:
Up to @tylerkelly13 
```suggestion
Custom colors cannot be enabled.
```

Reply:
```suggestion
The [custom color picker]({{site.baseurl}}/configure/content-appearance/#custom_colors) is not available for the `tablecellbackgroundcolor` toolbar button or menu item.
```